src/models/usersDao.js

```diff
@@ -0,0 +1,24 @@
+const dataSource = require("./dataSource");
+
+const signUp = async (email, password, points) => {
+  try {
+    await dataSource.query(
+      `INSERT INTO users(
+        email,
+        password,
+        points
+      )VALUES (?,?,?);
+      `,
+      [email, password, points]
+    );
+  } catch (err) {
+    console.log(err);
+    const error = new Error(`INVALID_DATA_INPUT`);
+    error.statusCode = 500;
```

Comment:
상태코드 500은 예외처리를 하지 않았을 때 나는 에러입니다. 따라서 500이 아닌 400번대 코드를 사용해 주세요.
```suggestion
    error.statusCode = 400;
```